src/api/app/models/bs_request.rb

```diff
@@ -38,6 +38,9 @@ class BsRequest < ApplicationRecord
   scope :from_project, ->(project_name) { where('bs_request_actions.source_project like ?', project_name) }
   scope :to_project, ->(project_name) { where('bs_request_actions.target_project like ?', project_name) }
 
+  scope :from_project_names, ->(project_names) { where(bs_request_actions: { source_project: project_names }) }
+  scope :to_project_names, ->(project_names) { where(bs_request_actions: { target_project: project_names }) }
```

Comment:
What's the difference to the scope above?

Reply:
The ones above take a single string and they query for a single partial match aka 'LIKE %project%'. The new ones take an array of complete project names and they query for perfect matches aka 'IN [project1, project2]'.
Combining both is possible but feels like overcomplicating them because it should always consider both matching criteria.